gradle.properties

```diff
@@ -35,4 +64,9 @@ devEnvWorkspace=workspace
 dbContainerName=CSDEpostgresql
 
 # R3 internal repository
-artifactoryContextUrl=https://staging.download.corda.net/maven/20ede3c6-29c0-11ed-966d-b7c36748b9f6-Beta1.0-HC00
+#artifactoryContextUrl=https://www.youtube.com
+artifactoryContextUrl=https://staging.download.corda.net/maven/20ede3c6-29c0-11ed-966d-b7c36748b9f6-Fox1.0-RC01

```

Comment:
Should we be merging these to the release brnach , which is the first branch anyone sees when looking at the repo? 

These URLs are intended for internal testing - seems like these woul dbe better suited to se set in a specific tag build for a given RC and leave a reasonable default in release/*

Reply:
I think I probably agree with you, we should change our strategy here.